scripts/build.js

```diff
@@ -14,7 +14,8 @@ const args = parseArgs(process.argv.slice(2), {
   try {
     const buildOptions = {
       entryPoints: [
-        'src/baklava.ts',
+        'src/index.ts',
+        'src/react/index.ts',
```

Comment:
Do we need to rename main file? Since our main usage approach is CDN, using library/component name as file name has some practical advantages. In the developer tools network tab, you can see more informational `baklava.js` instead of `index.js`. So I would prefer to keep `baklava.ts` and add `baklava-react.ts` instead of `react/index.ts`.

Reply:
okay. we will organize it as `baklava.ts` and `baklava-react.ts`.